centreon/src/Core/Domain/Configuration/Model/MetaService.php

```diff
@@ -30,70 +31,53 @@ class MetaService
                  CALCULTATION_TYPE_MINIMUM = 'minimum',
                  CALCULTATION_TYPE_MAXIMUM = 'maximum',
                  CALCULTATION_TYPE_SUM = 'sum';
-
     public const AVAILABLE_CALCULATION_TYPES = [
         self::CALCULTATION_TYPE_AVERAGE,
         self::CALCULTATION_TYPE_MAXIMUM,
         self::CALCULTATION_TYPE_MINIMUM,
         self::CALCULTATION_TYPE_SUM,
     ];
-
     public const DATA_SOURCE_GAUGE = 'gauge',
                  DATA_SOURCE_COUNTER = 'counter',
                  DATA_SOURCE_DERIVE = 'derive',
                  DATA_SOURCE_ABSOLUTE = 'absolute';
-
     public const AVAILABLE_DATA_SOURCE_TYPES = [
         self::DATA_SOURCE_ABSOLUTE,
         self::DATA_SOURCE_COUNTER,
         self::DATA_SOURCE_DERIVE,
         self::DATA_SOURCE_GAUGE,
     ];
-
     public const META_SELECT_MODE_LIST = 1;
     public const META_SELECT_MODE_SQL_REGEXP = 2;
-
     public const MAX_NAME_LENGTH = 254;
 
-    /**
-     * @var string|null
-     */
+    /** @var string|null */
     private $output;
 
-    /**
-     * @var string|null Search string to be used in a SQL LIKE query for service selection
-     */
+    /** @var string|null Search string to be used in a SQL LIKE query for service selection */
     private $regexpSearchServices;
 
-    /**
-     * @var string|null Select the metric to measure for meta service status.
-     */
+    /** @var string|null select the metric to measure for meta service status */
     private $metric;
 
-    /**
-     * @var int|null Absolute value for warning level (low threshold).
-     */
+    /** @var int|null absolute value for warning level (low threshold) */
     private $warningThreshold;
 
-    /**
-     * @var int|null Absolute value for critical level (high threshold).
-     */
+    /** @var int|null absolute value for critical level (high threshold) */
     private $criticalThreshold;
 
-    /**
-     * @var bool Indicates whether this Meta Service is enabled or not (TRUE by default)
-     */
+    /** @var bool Indicates whether this Meta Service is enabled or not (TRUE by default) */
     private $isActivated = true;
 
     /**
      * @param int $id
      * @param string $name
      * @param string $calculationType
      * @param int $metaSelectionMode Selection mode for services to be considered for this meta service.
-     *     0 - In service list mode, mark selected services in the options on meta service list.
-     *     1 - In SQL matching mode, specify a search string to be used in an SQL query.
+     *                               0 - In service list mode, mark selected services in the options on meta service list.
+     *                               1 - In SQL matching mode, specify a search string to be used in an SQL query.
```

Comment:
Not sure we should keep this rule 🤔
In addition, the 120-character limit per line is not respected.